__init__.py

```diff
@@ -782,10 +990,24 @@ def handle_change_alarm(self, _):
 
     def converse(self, utterances, lang="en-us"):
         """While an alarm is expired, check all utterances for Stop vocab."""
-        if has_expired_alarm(self.settings["alarm"]):
+        self.log.debug("Enter AlarmSkill converse utt:%s" % (utterances,))
+        if has_expired_alarm(self.settings["alarm"]) and utterances:
+            self.log.debug("AlarmSkill will consume the utterance")
             if utterances and self.voc_match(utterances[0], "StopBeeping"):
-                self._stop_expired_alarm()
-                return True  # and consume this phrase
+                self._stop_expired_alarm()  
+            elif self.voc_match(utterances[0], "Snooze"):
+                message = Message(
+                  "internal.snooze",
```

Comment:
a search for "internal.snooze" in this PR returns only this line of code.  If something outside this skill is handling this message, we should not name it "internal".  If it is not being handled anywhere, remove it.

Reply:
It is being handled by the snooze alarm method which is inside this skill which is why it is named internal. I'm afraid I don't understand the comment.

Reply:
Damn, I searched the PR, not the entire skill.  My bad.  Resolving.